src/classes/action/movie.vala

```diff
@@ -42,6 +42,40 @@ namespace pdfpc {
         public Window.Fullscreen window { get; set; }
     }
 
+    protected struct PlaybackOptions {
+        /**
+         * A flag to indicate whether the movie should start automatically.
+         */
+        bool autostart;
```

Comment:
Is this flag in preparation for the (non working) autoplay feature (as mentioned within the man page)

Reply:
The same structure is also used for the "href" style movie links, where autoplay does work (perhaps the main reason we still support it...)